### DISCUSSION TITLE
用一段时间后会变慢

### DISCUSSION THREAD

**Discussion by xxtyyq** [jxxghp/MoviePilot#2245] [问题咨询]
你好，我的mp每用三四天左右，就会变得特别慢，得重启才会没问题，是应该调整哪个设置吗？我是技术小白，麻烦指导，多谢多谢！

**Comment by dzplus** [comment]
哈喽我遇到了跟你一样的问题，我的解决方案是 去插件中心，下载一个叫做插件日志清理的插件，目前我已经跑了两天一切正常
<img width="297" alt="image" src="https://github.com/user-attachments/assets/f2e98699-bc25-46a3-8438-730d260a30d4">

**Reply by xxtyyq** [reply]
多谢多谢！原来是单独删除
